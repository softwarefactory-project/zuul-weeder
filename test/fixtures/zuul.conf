[connection gerrit]
driver=gerrit
server=managesf.sftests.com
canonical_hostname=sftests.com