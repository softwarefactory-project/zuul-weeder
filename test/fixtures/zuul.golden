Just
    [
        ( CanonicalProjectName
            ( ProviderName "sftests.com"
            , ProjectName "config"
            )
        ,
            [ PipelineT
            , JobT
            , SemaphoreT
            , ProjectT
            , ProjectTemplateT
            , NodesetT
            , SecretT
            , QueueT
            ]
        )
    ,
        ( CanonicalProjectName
            ( ProviderName "sftests.com"
            , ProjectName "project-config"
            )
        ,
            [ PipelineT
            , JobT
            , SemaphoreT
            , ProjectT
            , ProjectTemplateT
            , NodesetT
            , SecretT
            , QueueT
            ]
        )
    ,
        ( CanonicalProjectName
            ( ProviderName "sftests.com"
            , ProjectName "sf-jobs"
            )
        ,
            [ JobT
            , SemaphoreT
            , ProjectT
            , ProjectTemplateT
            , NodesetT
            , SecretT
            , QueueT
            ]
        )
    ,
        ( CanonicalProjectName
            ( ProviderName "sftests.com"
            , ProjectName "zuul-jobs"
            )
        , [ JobT ]
        )
    ,
        ( CanonicalProjectName
            ( ProviderName "sftests.com"
            , ProjectName "zuul-distro-jobs"
            )
        , []
        )
    ,
        ( CanonicalProjectName
            ( ProviderName "sftests.com"
            , ProjectName "no-include-proj"
            )
        , []
        )
    ]